CONTRIBUTING.md

```diff
@@ -63,6 +63,20 @@ Run the e2e tests by:
 yarn example test:e2e
 ```
 
+To build the [example app](/example/) in bare workflow run:
+
+```sh
+cd example/ && npx expo prebuild
+```
+
+Before compiling iOS in bare workflow remember to uncomment these lines in `example/ios/Podfile`:
+
+```ruby
+if !ENV['CI']
+  use_flipper!()
+end
+```
```

Comment:
We could automatically do this in the script

Reply:
Fixed in [145d4a3](https://github.com/react-navigation/react-navigation/pull/11020/commits/145d4a39101e0d04320edde9b105279a333fae6f)

I want to let you know this flow is a bit flaky - it has to fail at least once.

It runs `npx expo prebuild` and it fails after a while because it can't see flipper. The same script enables `use_flipper` right after that.

On the second run of the command, it works as expected. It kinda has to be like that because we can't do something in the middle of the `npx expo prebuild` command 😢

Reply:
Fixed in [c06a4c6](https://github.com/react-navigation/react-navigation/pull/11020/commits/c06a4c621b369ada479499f3684887a248cfb9c4)